src/tests/system/tests/test_default_debug_level.py

```diff
@@ -0,0 +1,196 @@
+        3. Check logs
+    :expectedresults:
+        1. Permissions changed successfully
+        2. SSSD fails to start
+        3. Fatal failures (level 0) and Critical failures (level 1) are in log file
+    :customerscenario: True
+    """
+    client.sssd.common.local()
+    client.sssd.default_domain = "local"
+    client.sssd.start(debug_level=None)
+
+    assert client.host.ssh.run("chmod 444 /etc/sssd/sssd.conf").rc == 0
+    assert client.sssd.restart(debug_level=None, raise_on_error=False, apply_config=False).rc != 0
+
+    log_str = client.fs.read("/var/log/sssd/sssd.log")
+    assert "0x0010" in log_str, "Fatal failures were not found in log"
+    assert "0x0020" in log_str, "Critical failures were not found in log"
+
+
+@pytest.mark.ticket(bz=1893159)
+@pytest.mark.topology(KnownTopology.Client)
+def test_default_debug_level__cannot_load_sssd_config(client: Client):
+    """
+    :title: default debug logs: Check default debug level 2
+    :setup:
+        1. Set 'domains' to 'non_existing_domain' in sssd section
+    :steps:
+        1. Start SSSD with default debug level
+        2. Check logs
+    :expectedresults:
+        1. SSSD failed to start
+        2. Correct error message is in log file
+    :customerscenario: True
+    """
+    client.sssd.sssd["domains"] = "non_existing_domain"
+    assert client.sssd.start(debug_level=None, raise_on_error=False).rc != 0
+    assert "SSSD couldn't load the configuration database" in client.fs.read("/var/log/sssd/sssd.log")
+
+
+@pytest.mark.ticket(bz=1893159)
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_default_debug_level__nonexisting_ldap_server(client: Client):
+    """
+    :title: default debug logs: Check default debug level 2
+        are printed with default log level
+    :setup:
+        1. Start sssd
+        2. Set ldap_uri to a non-existing ldap-server
+    :steps:
+        1. Check logs
+    :expectedresults:
+        1. Domain Logs should contain a log related to 'going offline'
+    :customerscenario: True
+    """
+    client.sssd.domain["ldap_uri"] = "ldap://typo"
+    client.sssd.start(debug_level=None, raise_on_error=False)
+
+    assert "Failed to connect, going offline" in client.fs.read(
+        f"/var/log/sssd/sssd_{client.sssd.default_domain}.log"
+    ), "Searched string was not found in logs"
+
+
+@pytest.mark.ticket(bz=1915319)
+@pytest.mark.topology(KnownTopology.Client)
+def test_default_debug_level__sbus(client: Client):
+    """
+    :title: Check SBUS code should not trigger failure
+        message during modules startup
+    :setup:
+        1. Start sssd
+        2. Check that debug_level is not in sssd.conf file
+    :steps:
+        1. Check logs
+    :expectedresults:
+        1. "Unable to remove key" is not in the logs
+    :customerscenario: True
+    """
+    client.sssd.common.local()
+    client.sssd.default_domain = "local"
+    client.sssd.start()
+    with pytest.raises(KeyError):
+        client.sssd.sssd["debug_level"]
```

Comment:
What exactly is this and why it is there?

Reply:
Basicaly, I forgot to correct this test. There should be `client.sssd.start(debug_level=None)`. This will cause that SSSD starts with default debug level.